x-pack/test/functional/apps/maps/full_screen_mode.js

```diff
@@ -10,11 +10,13 @@ export default function ({ getService, getPageObjects }) {
   const PageObjects = getPageObjects(['maps', 'common']);
   const retry = getService('retry');
   const esArchiver = getService('esArchiver');
+  const renderable = getService('renderable');
 
   describe('maps full screen mode', () => {
     before(async () => {
       await esArchiver.loadIfNeeded('maps/data');
       await PageObjects.maps.openNewMap();
+      await renderable.waitForRender();
```

Comment:
Would it make sense to move `await renderable.waitForRender()` into `PageObjects.maps.openNewMap()`? Would this improve stability elsewhere?

Reply:
Good point, I will give it a try.